### ISSUE TITLE
Why not divide the data by training set, validation set, test set, but only the training and validation set

### ISSUE THREAD

**Issue by HUCONGG** [CUHK-AIM-Group/U-KAN#9]
Why not divide the data by training set, validation set, test set, but only the training and validation set

**Comment by Docnoah** [comment]
good question! I have the same confusion

**Comment by yyydfff** [comment]
因为没有测试文件
